client/landing/jetpack-cloud/components/server-credentials-wizard-dialog/index.tsx

```diff
@@ -0,0 +1,103 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { translate } from 'i18n-calypso';
+import classnames from 'classnames';
+import { Dialog } from '@automattic/components';
+
+/**
+ * Internal dependencies
+ */
+import ExternalLink from 'components/external-link';
+import Gridicon from 'components/gridicon';
+import ServerCredentialsForm from 'landing/jetpack-cloud/components/server-credentials-form';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+interface Props {
+	onCloseDialog: Function;
+	showDialog: boolean;
+	siteId: number;
+	showServerCredentialStep: boolean;
+	children: React.ReactNode;
+	buttons?: React.ReactNode;
+	baseDialogClassName?: string;
+	title: i18nCalypso.TranslateResult;
+	titleClassName?: string;
+}
+
+const ServerCredentialsWizardDialog = ( {
+	onCloseDialog,
+	showDialog,
+	siteId,
+	showServerCredentialStep,
+	buttons,
+	title,
+	titleClassName,
+	baseDialogClassName,
+	children,
+}: Props ) => {
+	return (
+		<Dialog
+			additionalClassNames={ classnames( 'server-credentials-wizard-dialog', baseDialogClassName ) }
+			isVisible={ showDialog }
+			buttons={ showServerCredentialStep ? undefined : buttons }
+			onClose={ onCloseDialog }
+		>
+			<h1 className={ classnames( 'server-credentials-wizard-dialog__header', titleClassName ) }>
+				{ title }
+			</h1>
+			{ showServerCredentialStep && (
+				<>
+					<h3 className="server-credentials-wizard-dialog__subheader">
+						{ translate( 'You have selected to fix all discovered threats' ) }
```

Comment:
> You have selected to fix all discovered threats

In my testing, I saw this copy whether I was fixing one threat or all of them. Do we want to differentiate between these two cases?

Reply:
Thanks for catching this! How would you name that piece of copy? Subtitle?

Reply:
Would you say that `You have selected to fix a threat` is good option for the single case?

Reply:
I think `You have selected to fix a discovered threat` works, and it would be grammatically consistent with the "all threats" copy.

If you want to go the extra mile, I like the idea of "You have selected to fix the following threat:" followed by a short summary of the selected threat.

Reply:
Copy updated. I think I'll leave the option of adding info about threat for later because we would have to update the UI in several places.

Thanks Matt!